src/statmodels.jl

```diff
@@ -93,6 +146,20 @@ Return the variance-covariance matrix for the coefficients of the model.
 """
 vcov(obj::StatisticalModel) = error("vcov is not defined for $(typeof(obj)).")
 
+"""
+    weights(obj::StatisticalModel)
+
+Return the weights used in the model.
+"""
+weights(obj::StatisticalModel) = error("weights is not defined for $(typeof(obj)).")
+
+"""
+    isfitted(obj::StatisticalModel)
+
+Return Boolean indicator of whether the model has been fitted.
```

Comment:
Maybe just "Indicate whether the model has been fitted."

Please check the other docstrings, as some say "the"/"a", and some have no determinant. Some also repeat "the" twice.